src/mcp/tools/project/get_sdk_config.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint camelcase: 0 */
```

Comment:
Dumb question - why do we need to not use camelcase names for these tools?

Reply:
Technically we could make all of this stuff camelCase, but:

1. Models are trained heavily on Python tools and `snake_case` seems to be the most common.
2. I thought it was better to have the exports be named consistently with the underlying tool named consistently with the file...but it's really a choice to do consistency and disable the lint error over inconsistency but lint passes.

TBH I hate ESLint 😆